x/tunnel/keeper/msg_server.go

```diff
@@ -69,6 +72,11 @@ func (k msgServer) CreateTunnel(
 		if !found {
 			return nil, types.ErrInvalidChannelID
 		}
+	case *types.RouterRoute:
+		_, found := k.channelKeeper.GetChannel(ctx, ibctransfertypes.PortID, routerIBCChannel)
+		if !found {
+			return nil, types.ErrInvalidChannelID
+		}
```

Comment:
don't think need to check it for Router.  it should always be available.
Also, if it's not, it can't be reliable and it can be changed later from params.

wdyt?.